python/lsst/ctrl/mpexec/cli/opt/options.py

```diff
@@ -310,17 +310,17 @@
     callback=split_commas,
     help=unwrap(
         """Dump various info to standard output. Possible items are:
-                                            `config`, `config=[Task::]<PATTERN>` or
-                                            `config=[Task::]<PATTERN>:NOIGNORECASE` to dump configuration
+                                            ``config``, ``config=[Task::]<PATTERN>`` or
+                                            ``config=[Task::]<PATTERN>:NOIGNORECASE`` to dump configuration
                                             fields possibly matching given pattern and/or task label;
-                                            `history=<FIELD>` to dump configuration history for a field, field
-                                            name is specified as [Task::]<PATTERN>; `dump-config`,
-                                            `dump-config=Task` to dump complete configuration for a task given
-                                            its label or all tasks; `pipeline` to show pipeline composition;
-                                            `graph` to show information about quanta; `workflow` to show
-                                            information about quanta and their dependency; `tasks` to show
-                                            task composition; `uri` to show predicted dataset URIs of
-                                            quanta"""
+                                            ``history=<FIELD>`` to dump configuration history for a field,
+                                            field name is specified as ``[Task::]<PATTERN>;`` ``dump-config``,
+                                            ``dump-config=Task`` to dump complete configuration for a task
+                                            given its label or all tasks; ``pipeline`` to show pipeline
+                                            composition; ``graph`` to show information about quanta;
+                                            ``workflow`` to show information about quanta and their
+                                            dependency; ``tasks`` to show task composition; ``uri`` to show
+                                            predicted dataset URIs of quanta"""
```

Comment:
Maybe de-indent the whole thing?

Reply:
Yes, I was trying to work out why it was all like this and failing 😄

Reply:
I think that it was initially indented to an opening `(` but then black wrapped the initial line but did not de-indent other lines.